wesnoth-utbs.po

```diff
@@ -16071,9 +16153,11 @@ msgstr "osvěžená"
 msgid "refreshed"
 msgstr "osvěžený"
 
+# Originální řetězec je nějaký divný.
```

Comment:
mohlo by to být v pořádku - %d se u programátorů nahradí decimální hodnotou předanou ve funkci
```suggestion
```